app/src/main/kotlin/co/netguru/baby/monitor/client/feature/babynotification/BabyMonitorMessagingService.kt

```diff
@@ -47,6 +47,11 @@ class BabyMonitorMessagingService : FirebaseMessagingService() {
                 message.data
             )
             NotificationType.LOW_BATTERY_NOTIFICATION -> handleLowBatteryNotification(message.data)
+            NotificationType.CRY_NOTIFICATION_WITH_FEEDBACK_REQUEST,
+            NotificationType.NOISE_NOTIFICATION_WITH_FEEDBACK_REQUEST -> {
+                // TODO handle notification with feedback request
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/7c069c26782df05dac8a097aa956e351ad95f920/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/babynotification/BabyMonitorMessagingService.kt#L52"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>